client/my-sites/plans-grid/components/features-grid/index.tsx

```diff
@@ -364,56 +365,59 @@ class FeaturesGrid extends Component< FeaturesGridType > {
 			handleUpgradeClick,
 		} = this.props;
 
-		return renderedGridPlans.map( ( { planSlug, availableForPurchase } ) => {
-			const classes = classNames( 'plan-features-2023-grid__table-item', 'is-top-buttons' );
-
-			// Leaving it `undefined` makes it use the default label
-			let buttonText;
-
-			if (
-				isWooExpressMediumPlan( planSlug ) &&
-				! isWooExpressMediumPlan( currentSitePlanSlug || '' )
-			) {
-				buttonText = translate( 'Get Performance', { textOnly: true } );
-			} else if (
-				isWooExpressSmallPlan( planSlug ) &&
-				! isWooExpressSmallPlan( currentSitePlanSlug || '' )
-			) {
-				buttonText = translate( 'Get Essential', { textOnly: true } );
-			}
+		return renderedGridPlans.map(
+			( { planSlug, availableForPurchase, features: { storageOptions } } ) => {
```

Comment:
**Note:**

L368-420 are caused by auto-formatting. The only meaningful change is that we destructure the `storageOptions` parameter